src/views/Account/Account.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect } from "react";
+import { Outlet, useNavigate } from "react-router-dom";
+
+import ProfileSideBar from "components/organisms/ProfileSideBar/ProfileSideBar";
+import MainLayout from "layouts/MainLayout";
+
+import * as S from "./styles";
+import { useUser } from "lib/auth";
+
+const Account = () => {
+  const navigate = useNavigate();
+
+  const { user, isLoading } = useUser();
+
+  useEffect(() => {
+    if (!isLoading && !user) {
+      navigate("/login");
+    }
+
+    //eslint-disable-next-line
```

Comment:
why?

Reply:
Because it then tells me to add navigate to the dependency array